.travis.yml

```diff
@@ -1,11 +0,0 @@
-version: ~> 1.0
-
-services:
-  - docker
```

Comment:
what's the equivalent for this in GitHub Actions? 🤔

Reply:
I don't know if you've seen this migration guide that GitHub themselves wrote to migrate from Travis -> GitHub Actions:
https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions

---

I think I found something relevant here:

https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-nodejs#running-on-a-different-operating-system which says:

> You can also run jobs in Docker containers, or you can provide a self-hosted runner that runs on your own infrastructure. For more information, see "Workflow syntax for GitHub Actions."

That quoted section links to this: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on if you search for the string `docker` on this page, you'll get 51 hits, maybe something in that page can help you get further.

I'm not going to recommend any course of action, as I'm not qualified to do so... :smile:

Reply:
thanks for digging this up! I've looked through it but didn't find anything that would setup the `docker` binary. For now we will run the tests only on ubuntu which seems to have it installed